### DISCUSSION TITLE
Error 400 - Bad Request when attempting to communicate with Windows agent

### DISCUSSION THREAD

**Discussion by Darryl-One-Degree** [amidaware/tacticalrmm#1185] [Q&A]
Morning All

I am getting Error 400 Bad Request when I try and communicate with a deployed agent. What can I look at in order to resolve this issue ?

**Comment by dinger1986** [comment]
Pre-filling the relevant answers to these questions will also help us help you:

1. Install type? (Standard/Docker)
  Did you deviate IN ANY WAY from these instructions? https://docs.tacticalrmm.com/install_server/
  Did you deviate IN ANY WAY from these instructions? https://docs.tacticalrmm.com/install_docker/
2. Where is the server? (VPS/onprem)
3. New/old install? Rough age of TRMM server (days/weeks/months)?

Server Install Specific questions:
4. What OS/version is TRMM installed on.
5. Using a real domain?
6. Did letsencrypt finalize and work
7. Have you looked at the troubleshooting steps? https://docs.tacticalrmm.com/troubleshooting/
8. What kind of ssl certs? Let's Encrypt, or purchased (you're not trying to make self-signed work right?)
9. Check Expiry date of your certificates in the browser (at https://rmm.example.com/ )

Network Troubleshooting
10. Are you using a proxy?
11. Are you a wizard? See https://docs.tacticalrmm.com/unsupported_guidelines/
  If so, what's in the network between agent and server?

Agent Troubleshooting
12. Is there ANY 3rd party Antivirus installed on the computer?
13. Is there any network based filtering/AV filtering?

Asking yourself questions like: When did it go from working to broken? What changed during that time? This will help you.

**Comment by silversword411** [comment]
https://docs.tacticalrmm.com/troubleshooting/#cant-login-on-server-after-first-setup

What do you mean "communicate"?
